apps/trading/components/rewards-container/active-rewards.tsx

```diff
@@ -568,7 +666,18 @@ const StakingRewardCard = ({
                 className="flex items-center gap-1"
                 data-testid="staking-requirement"
               >
-                {formatNumber(1, 2)}
+                {!stakeAvailable ? (
+                  '1.00'
+                ) : stakeAvailable > 1 ? (
+                  <Tick />
+                ) : (
+                  <Cross />
+                )}
+                {addDecimalsFormatNumber(
+                  stakeAvailable?.toString() || '0',
+                  18, // vega asset decimals
```

Comment:
can we not use vegaAsset here?

Reply:
however we refer to staked vega and  we get an 18 dp value

Reply:
sorry I missunderstood the comment. changed to `addDecimalsFormatNumber(stakeAvailable?.toString() || '0',vegaAsset?.decimals || 18)`